src/fontra/views/editor/panel-glyph-search.js

```diff
@@ -17,6 +19,49 @@ export default class GlyphSearchPanel extends Panel {
     }
   `;
 
+  setup() {
+    this.editorController.glyphsSearch =
```

Comment:
Why do you assign to editorController here? I believe you meant simple "this".

Reply:
Or else I don't understand what's goin on here. But you should not assign to editorController from outside of editorController itself.